apps/pr-deploy-site/just.config.js

```diff
@@ -32,7 +35,7 @@ repoDeps.forEach(dep => {
 preset();
 
 task(
-  'build',
+  'bundle',
```

Comment:
Maybe a silly question but why is this `bundle` now?

Reply:
Is there a `yarn start` that will fail anywhere if this command is not executed? I interpreted this command as a 'bundle' step of sorts before `pr-deploy-site` is deployed.

Reply:
There also isn't any TS or typechecking that I see here, so I'm not sure this is really a build command in any case.

Reply:
After further investigation, this command actually needs to copy the `bundle` output of other packages, and additionally, I needed to make `bundle` run in `--stream` rather than `--parallel` so as not to break this task. For these reasons it can't run in the `build` phase.